ext/libclementine-tagreader/tagreader.cpp

```diff
@@ -130,6 +130,43 @@ const char* kASF_OriginalDate_ID = "WM/OriginalReleaseTime";
 const char* kASF_OriginalYear_ID = "WM/OriginalReleaseYear";
 }  // namespace
 
+// Helpers for GuessArtistAndTitle()
+static QString withoutExtension(const QString  s) {
+  if (s.isEmpty()) return s;
+  const int i = s.lastIndexOf('.');
+  if (i  < 0) return s;
+  return s.left(i);
+}
+
+static inline void changeUnderscores(QString &s) {
```

Comment:
* Remove `static inline`
* Change name to `ReplaceUnderscoresWithSpaces`
* Change arg to `const QString& s`
* Return a new `QString`